io_scene_niftools/properties/scene.py

```diff
@@ -94,6 +93,15 @@ class Scene(PropertyGroup):
         default='NONE',
         update=update_version_from_game)
 
+    rootnode: EnumProperty(
```

Comment:
Should this be at scene level?

Reply:
Should we have a reference to an object which would used as the root node?

Reply:
> Should we have a reference to an object which would used as the root node?

I think then we just delegate the problem. It's better to tag root objects (eg. those without a parent are automatically root objects)

Reply:
I was more thinking in the case that we have multiple objects tags as root object or do we allow that.